microsoft-edge/hosting/webview2/ReleaseNotes.md

```diff
@@ -29,7 +29,7 @@ This SDK is our official Win32 C++ Beta version, which incorporates several feat
 *   Added support for `Windows Server 2008 R2`, `Windows Server 2012/2012 R2`, and `Windows Server 2016`
 *   Added [NotifyParentWindowPositionChanged](reference/icorewebview2host.md#notifyparentwindowpositionchanged) which allows for the status bar to follow the window in windowed mode. This will also need to be implemented in windowless mode in order for accessibility features to work.
 *   Added [AreRemoteObjectsAllowed](reference/icorewebview2settings.md#get_areremoteobjectsallowed) setting to globally control whether a page can be accessed by any remote objects. By default, AreRemoteObjectsAllowed is enabled, meaning remote objects added by [AddRemoteObject](reference/icorewebview2.md#addremoteobject) are accessible from the page. When AreRemoteObjectsAllowed is disabled, the objects will not be accessible from the page. Changes will be applied on the next navigation event.
-*   Added [IsZoomControlEnabled](reference/icorewebview2settings.md#get_iszoomcontrolenabled) setting. By default, zoom is enabled within the WebView. Setting IsZoomControlEnabled to False disables zooming.  ([#95](https://github.com/MicrosoftEdge/WebViewFeedback/issues/95)).  
+*   Added [IsZoomControlEnabled](reference/icorewebview2settings.md#get_iszoomcontrolenabled) setting. By default, zoom is enabled within the WebView. Setting IsZoomControlEnabled to False disables zooming. Aside from the setting, ZoomView UI is hidden for WebView. ([#95](https://github.com/MicrosoftEdge/WebViewFeedback/issues/95)).  
```

Comment:
I'd put the zoomview as a separate item from the setting.

Reply:
Done :)